internal/consumer/cron.go

```diff
@@ -0,0 +1,33 @@
+package consumer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/model"
+
+	"github.com/robfig/cron/v3"
+	"go.uber.org/zap"
+)
+
+func ConfigureCron(ctx context.Context, logger *zap.Logger, config *model.Configuration) error {
+	c := cron.New()
+
+	var err error
+	storyProcessing := func() {
+		service, _ := NewService(logger, &config.Consumer, nil)
```

Comment:
How come an anonymous func is being used here?

Reply:
I used an anonymous function to make it easier to call the functionality the first time as well as within the cron AddFunc method. 

If there is a better practice to handle this scenario, please shout